content/influxdb/cloud-dedicated/reference/client-libraries/v3/python.md

```diff
@@ -14,43 +14,36 @@ aliases:
   - /influxdb/cloud-dedicated/reference/client-libraries/v3/pyinflux3/
 related:
   - /influxdb/cloud-dedicated/query-data/execute-queries/troubleshoot/
-list_code_example: >
-  <!-- Import for tests and hide from users.
-  ```python
-  import os
-  ```
-  -->
-  <!--pytest-codeblocks:cont-->
```

Comment:
If you just remove the comments and commented lines, it should fix the list page display--that's the quick fix.

As a more strategic fix, I like your idea, but...
- it would also require changing the section (`_index`) page template to support it.
- the page that includes the example would need to hide it from displaying

I'm fine with a quick fix. For the "strategic" fix, my preference would probably be to change the parsing/rendering of the existing `list_code_example` frontmatter field to support the test setup comments.

Reply:
...just thinking out loud... a better "fix" might be to have the test runner inject

  ```python
  import os
  ```

for each example before running it - that eliminates the need for alot of pytest comments in the pages.